src/components/Comments.js

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getComments } from "../utils/api";
+
+function Comments() {
+  const [comments, setComments] = useState([]);
+  const [loadingComments, setLoadingComments] = useState(true);
+
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    getComments(article_id).then((retreivedComments) => {
```

Comment:
you need to set loading to be true at the start of the useEffect, otherwise if you switch pages it won't render the loading message